CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 * [FEATURE] PromQL: Add experimental `info` function. Experimental functions are disabled by default, but can be enabled setting `-querier.promql-experimental-functions-enabled=true` in the query-frontend and querier. #9879
 * [FEATURE] Distributor: Support promotion of OTel resource attributes to labels. #8271
 * [FEATURE] Querier: Add experimental `double_exponential_smoothing` PromQL function. Experimental functions are disabled by default, but can be enabled by setting `-querier.promql-experimental-functions-enabled=true` in the query-frontend and querier. #9844
+* [FEATURE] Distributor: Add experimental `memberlist` KV store for ha_tracker. It can be enabled using the `-distributor.ha-tracker.kvstore.store` flag. Memberlist parameters can be configured via the `-memberlist-*` flags. #10054
```

Comment:
```suggestion
* [FEATURE] Distributor: Add experimental `memberlist` KV store for ha_tracker. You can enable it using the `-distributor.ha-tracker.kvstore.store` flag. You can configure Memberlist parameters via the `-memberlist-*` flags. #10054
```

Reply:
We can use active voice to address the user directly.

Reply:
Thank you . Addressed at [94e86e8](https://github.com/grafana/mimir/pull/10054/commits/94e86e86730024d4fbf3fab2c8d47f15cd470bfc)